code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -86,8 +86,7 @@
 	var/obj/machinery/nuclearbomb/existing = locate("syndienuke") in GLOB.nuke_list
 	if(existing)
 		qdel(src)
-		throw EXCEPTION("Attempted to spawn a syndicate nuke while one already exists at [existing.loc.x],[existing.loc.y],[existing.loc.z]")
-		return 0
+		CRASH("Attempted to spawn a syndicate nuke while one already exists at [existing.loc.x],[existing.loc.y],[existing.loc.z]")
```

Comment:
yeah but it was explicitly returning 0 before, not .
It's almost certainly fine, but you should probably check if it matters.